src/fty_shm_cleanup.cc

```diff
@@ -62,6 +63,11 @@ bool clean_outdated_data(std::string filename) {
   time_t now, ttl;
   FILE* file = fopen(filename.c_str(), "r");
 
+  if (!file) {
+    log_error("Cannot open %s for cleaning", filename.c_str());
+    return -1;
```

Comment:
Is this a `bool` function? Does `return -1` have it return a `true`?

Reply:
Comments above the declaration say it is a tri-state so should be int. And the one consumer I found so far does not care.

Reply:
true, I've blindly made the code with things around... which were wrong :D
will fix the rest in the next commit